exercises/secret-handshake/example.js

```diff
@@ -12,13 +12,13 @@ function SecretHandshake(handshake) {
     return this.shakeWith;
   };
 
-  this.calculateHandshake = function (handshake) {
+  this.calculateHandshake = function (handshakeInner) {
```

Comment:
Why the name change here?

Reply:
Definitely, happy to provide some context! For this change, I was getting the no-shadow error from the linter: /*error 'handshake' is already declared in the upper scope.*/ Changing the input name to handshakeInner I was able to replicate the results and resolve the error.

Reply:
The linter is right- `handshake` _is_ already declared.  So you don't need to pass it in to the inner function `this.calculateHandshake`, because it's available already when `this.calculateHandshake` is declared.

Reply:
Thanks for your help on this! Was overthinking it.